README.md

```diff
@@ -5,6 +5,10 @@ Manuals Frontend
 
 Front-end app for the manuals format on GOV.UK
 
+## Live examples
+
+- [gov.uk/guidance/content-design](https://www.gov.uk/guidance/content-design)
```

Comment:
I think its worth adding an HMRC manual example as they do differ in several ways: https://www.gov.uk/hmrc-internal-manuals/vat-government-and-public-bodies